tensorflow/lite/mutable_op_resolver_test.cc

```diff
@@ -112,8 +122,34 @@ TEST(MutableOpResolverTest, FindCustomOp) {
   EXPECT_EQ(found_registration->builtin_code, BuiltinOperator_CUSTOM);
   EXPECT_TRUE(found_registration->invoke == DummyInvoke);
   EXPECT_EQ(found_registration->version, 1);
-  // TODO(ycling): The `custom_name` in TfLiteRegistration isn't properly
```

Comment:
Hmm, this CL isn't really addressing this. I think the idea is to initialize the ".custom_name" field of the registration, and validate that.

Reply:
@jdduke , got it , kindly check the new TC update, i have followed as per your suggestion.

Regards
Amit